Updating for italics and quotes consistency, S-Z

bergerhoffer opened the pull request:
Updated letter files S-Z for consistent usage of quotes and italics.

Preview: http://file.rdu.redhat.com/~ahoffer/2022/main-italics.html

Issue #173

bergerhoffer commented:
@redhat-documentation/ccs-style-council For review please!

ncbaratta reviewed (approved):
lgtm

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:
```diff
@@ -139,7 +139,7 @@
 [[throughput]]
 ==== image:images/yes.png[yes] throughput (noun)
-*Description*: "Throughput" is the amount of data transferred from one place to another or processed in a specified amount of time. Data transfer rates for disk drives and networks are measured in terms of throughput. Typically, throughput is measured in kbps, Mbps, or Gbps. See the _IBM Style Guide_ for more information about using measurements and abbreviations.
+*Description*: _Throughput_ is the amount of data transferred from one place to another or processed in a specified amount of time. Data transfer rates for disk drives and networks are measured in terms of throughput. Typically, throughput is measured in kbps, Mbps, or Gbps. See the _IBM Style Guide_ for more information about using measurements and abbreviations.
```
Probably out of scope, but I'll mark as optional: should kbps, Mbps, or Gbps be surrounded by backticks?

bergerhoffer replied:
I don't necessarily think so, so I'll leave it as is.

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:
```diff
@@ -209,7 +209,7 @@ You can use a TGT to request further access, and provide the user with a Single
 [[tls]]
 ==== image:images/yes.png[yes] TLS (noun)
-*Description*: TLS is an initialism for Transport Layer Security (TLS), and it is the successor to the Secure Sockets Layer (SSL) protocol. Do not expand the abbreviation on the first usage.
+*Description*: _TLS_ is an initialism for "Transport Layer Security (TLS)", and it is the successor to the Secure Sockets Layer (SSL) protocol. Do not expand the abbreviation on the first usage.
```
Would a statement referring to use as an initialism or an abbreviation refer to the "use" of a term?

bergerhoffer replied:
That's a good question. I'm not really sure, but I think it could make sense in that way. I'll update throughout.

bergerhoffer replied:
@dfitzmau I updated so that anywhere I say that something is an acronym/initialism/abbreviation, it uses quotes. Can you check your open PRs to make sure you're following the same too?

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:
```diff
@@ -265,7 +265,7 @@ A client requests a certificate from a web server. On receiving the certificate,
 [[transactions]]
 ==== image:images/yes.png[yes] transactions subsystem (noun)
-*Description*: In Red Hat JBoss Enterprise Application Platform, the "transactions" subsystem is used to configure options in the Transaction Manager. Write in lowercase in general text. Use "Transactions subsystem" when referring to the transactions subsystem in titles and headings.
+*Description*: In Red Hat JBoss Enterprise Application Platform, the _transactions_ subsystem is used to configure options in the Transaction Manager. Write in lowercase in general text. Use "Transactions subsystem" when referring to the transactions subsystem in titles and headings.
```
_transactions_ subsystem_ to fit the title of the entry? 

Consider surrounding the last `transactions` entry in monospace to align with highlighting conventions for component names.

bergerhoffer replied:
I followed your lead in your PR #217, which didn't include "subsystem" in the italics. If you want to do that, we should do it throughout all of our PRs. wdyt?

dfitzmau replied:
Yeah, I realized on the latest changes I made the title mentions the word "subsystem", so maybe placing the word in italics also would help? I could apply this to any changes that I've made also.

bergerhoffer replied:
Sounds good, I'll make the changes in my files as well!

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:
```diff
@@ -313,7 +313,7 @@ A web server uses its public key to obtain a certificate from a trusted CA. The
 [[ttl]]
 ==== image:images/yes.png[yes] TTL (noun)
-*Description*: "TTL" is an abbreviation for "time to live" (noun) and "time-to-live" (adjective). The abbreviation is always in uppercase letters.
+*Description*: _TTL_ is an abbreviation for "time to live" (noun) and "time-to-live" (adjective). The abbreviation is always in uppercase letters.
```
"TTL", because of abbreviation reference?

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc:
```diff
@@ -37,7 +37,7 @@ Use lowercase "v" and uppercase "CPU".
 [[verb]]
 ==== image:images/yes.png[yes] verb (noun)
-*Description*: In Red Hat OpenShift, a verb is a get, list, create, or update operation.
+*Description*: In Red Hat OpenShift, a _verb_ is a get, list, create, or update operation.
```
"verb" as it refers to language use?

bergerhoffer replied:
I don't think it does, it's talking about the technical concept of it, which is a "get", "list", "create", or "update" operation. I'm thinking that it would help if these items were in backticks probably. I can update to add those.

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc:
```diff
@@ -197,7 +197,7 @@ Although the _IBM Style Guide_ recommends using "diskette drive" instead of "flo
 [[virtualized-guest]]
 ==== image:images/caution.png[with caution] virtualized guest (noun)
-*Description*: A "virtualized guest" is a virtual machine (VM). Use virtualized guest only when comparing a "fully virtualized guest" with a "paravirtualized guest".
+*Description*: A _virtualized guest_ is a virtual machine (VM). Use virtualized guest only when comparing a _fully virtualized guest_ with a _paravirtualized guest_.
```
Use "virtualized guest" only when

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc:
```diff
@@ -233,7 +233,7 @@ Always use "VM Portal" and capitalize the product name.
 [[vnic]]
 ==== image:images/yes.png[yes] vNIC (noun)
-*Description*: "vNIC" is an abbreviation for virtual network interface controller. Use lowercase v and uppercase NIC for the abbreviation, but all lowercase for the expansion, except at the beginning of a sentence.
+*Description*: _vNIC_ is an abbreviation for "virtual network interface controller". Use lowercase v and uppercase NIC for the abbreviation, but all lowercase for the expansion, except at the beginning of a sentence.
```
se lowercase "v" and uppercase "NIC" for the abbreviation, but all lowercase for the expansion, except at the beginning of a sentence.

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc:
```diff
@@ -23,7 +23,7 @@
 [[wca]]
 ==== image:images/yes.png[yes] WCA (noun)
-*Description*: "WCA" is an abbreviation for web clipping application, which is an application that allows users to extract static information from a web server and load that data onto a web-enabled PDA. WCAs are also called query applications.
+*Description*: _WCA_ is an abbreviation for "web clipping application", which is an application that allows users to extract static information from a web server and load that data onto a web-enabled PDA. WCAs are also called query applications.
```
WCAs are also called "query applications".

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc:
```diff
@@ -149,7 +149,7 @@
 [[window-maker]]
 ==== image:images/yes.png[yes] Window Maker (noun)
-*Description*: "Window Maker" is a window manager for the X Window System. Do not combine Window Maker into one word or hyphenate the two words.
+*Description*: _Window Maker_ is a window manager for the X Window System. Do not combine Window Maker into one word or hyphenate the two words.
```
Do not combine "Window Maker" into one word or hyphenate the two words.
Write Preview

dfitzmau commented:
> Updated letter files S-Z for consistent usage of quotes and italics.
> 
> Preview: http://file.rdu.redhat.com/~ahoffer/2022/main-italics.html
> 
> Issue #173

Hi @bergerhoffer . I added some comments inline. Please feel free to ignore some comments; probably related to my misinterpretation of things.

bergerhoffer commented:
@dfitzmau Thanks for the feedback, I've updated per your suggestions, or otherwise replied!

bergerhoffer commented:
@dfitzmau So I updated the italics to go around each subsystem like _<name> subsystem_ for all my letters in this PR.

But then in doing so, I realized that I missed updating u :shrug: So did u.adoc and pushed those updates too. Here's the diff if it helps to see the new things: https://github.com/redhat-documentation/supplementary-style-guide/compare/f1985a9206c59b9e572f29d59181699f8a9c862c..d02e08aa44ffbce7ec9ab89210c29aa968865bfe

bergerhoffer commented:
@dfitzmau Can you PTAL at the updates when you get a chance?

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/u.adoc:
```diff
@@ -47,7 +47,7 @@
 [[undertow]]
 ==== image:images/yes.png[yes] undertow subsystem (noun)
-*Description*: In Red Hat JBoss Enterprise Application Platform, the "undertow" subsystem is used to configure the JBoss EAP web server and servlet container settings. Write in lowercase in general text. Use "Undertow subsystem" when referring to the undertow subsystem in titles and headings. See the xref:webhttp-undertow[WebHTTP - Undertow] entry for the correct usage when referring to the under ...
+*Description*: In Red Hat JBoss Enterprise Application Platform, the _undertow subsystem_ is used to configure the JBoss EAP web server and servlet container settings. Write in lowercase in general text. Use "Undertow subsystem" when referring to the undertow subsystem in titles and headings. See the xref:webhttp-undertow[WebHTTP - Undertow] entry for the correct usage when referring to the under ...
```
Hi @bergerhoffer . On products that I've worked on to date, I generally surround a subsystem name in backticks when referring to it in a general way, because it's an established component of a product. For example: 

> when referring to the `undertow` subsystem in titles and headings

WDYT?

bergerhoffer replied:
Yep that works! Let's just make sure it gets done in the other letters' PRs too. I'll update throughout here

bergerhoffer commented:
@dfitzmau Updated - ready for re-review

dfitzmau commented:
> @dfitzmau So I updated the italics to go around each subsystem like _ subsystem_ for all my letters in this PR.
> 
> But then in doing so, I realized that I missed updating u shrug So did u.adoc and pushed those updates too. Here's the diff if it helps to see the new things: https://github.com/redhat-documentation/supplementary-style-guide/compare/f1985a9206c59b9e572f29d59181699f8a9c862c..d02e08aa44ffbce7ec9ab89210c29aa968865bfe

Thanks for providing the diff, @bergerhoffer . Might I ask the best way to submit such links for any PRs that I create?

dfitzmau commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:
```diff
@@ -302,7 +302,7 @@ A web server uses its public key to obtain a certificate from a trusted CA. The
 [[truth-maintenance-system]]
 ==== image:images/yes.png[yes] truth maintenance system (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, a "truth maintenance system" (TMS) refers to the ability of the inference engine to enforce truthfulness when applying rules. The truth maintenance system uses the mechanism of truth maintenance to efficiently handle the inferred information from rules. It provides justified reasoning for each and every action taken by the inferenc ...
+*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, a _truth maintenance system (TMS)_ refers to the ability of the inference engine to enforce truthfulness when applying rules. The truth maintenance system uses the mechanism of truth maintenance to efficiently handle the inferred information from rules. It provides justified reasoning for each and every action taken by the inferenc ...
```
```suggestion
*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, a _truth maintenance system (TMS)_ refers to the ability of the inference engine to enforce truthfulness when applying rules. The `truth maintenance` system uses the mechanism of truth maintenance to efficiently handle the inferred information from rules. It provides justified reasoning for each and every action taken by the inference engine and validates the conclusions of the engine. If the inference engine asserts data as a result of firing a rule, the engine uses the truth maintenance to justify the assertion.
```

bergerhoffer replied:
@dfitzmau Are you sure about this one? I searched the docs for this and in this doc that came up, I don't see "truth maintenance" in backticks at all: https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.7/html/decision_engine_in_red_hat_process_automation_manager/inference-and-truth-maintenance_decision-engine

(And same question as above comment - this is "system", not an EAP "subsystem")

dfitzmau commented:
> @dfitzmau Updated - ready for re-review

Thanks, @bergerhoffer . I added some minor comments inline.

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc:
```diff
@@ -14,7 +14,7 @@
 [[target-kernel]]
 ==== image:images/yes.png[yes] target kernel (noun)
-*Description*: In Red Hat Enterprise Linux, the target kernel is the kernel of the target system. This is the kernel that loads and runs the instrumentation module.
+*Description*: In Red Hat Enterprise Linux, the _target kernel_ is the kernel of the target system. This is the kernel that loads and runs the instrumentation module.
```
@dfitzmau Are you sure that this is something that needs to be in backticks? I just searched for it and a doc that came up (https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/systemtap_beginners_guide/cross-compiling) doesn't seem to put "target" in backticks. It italicized the whole thing, so I don't think this backtick would be correct.

(And this is "system", not an EAP "subsystem")

dfitzmau replied:
My bad. I thought these entries said "subsystem". Sorry, I've being jumping in and out of release stuff. PR looks good to me. I'll approve. Thanks for your patience.

inoxx03 commented:
@redhat-documentation/ccs-style-council please review and provide an ACK if. I'll review and ack this, too,  butr we need at least 2 more acks to merge this. Thank you:)

bburt-rh commented:
lgtm

bburt-rh commented:
@inoxx03 We have 2 approvals. Just need yours as well, and this is good to merge

inoxx03 commented:
@bburt-rh Sure thing! I'll ack #217, the this one, and I'll merge them both:)

bergerhoffer commented:
@bredamc Actually this one is on me. There's a discussion in chat about changing how we introduced terms that were abbreviations, so I need to update this PR to reflect that. Will do that in the next day or so!

bergerhoffer commented:
Okay, I updated to adjust the abbreviations to use italics when introducing the term, and quotes when saying what it's an abbreviation for. Also updated in letters A-F, which had already been merged for the italics/quotes updates.

Ready for re-review!

Here is the diff of changes: https://github.com/redhat-documentation/supplementary-style-guide/compare/e351ff9a76980a27037b0f507a7e3e5340b23148..923aaf08be970e45348292fb7a49218b5153c521

@dfitzmau This reminded me that you asked how to get that link ^ and I forgot to reply to that. If you look in the history of this PR, you can see where it says "@bergerhoffer force-pushed the updating-italics-quotes branch from e351ff9 to 923aaf0". If you click "force-pushed" in that line, it's a link that takes you to a comparison of these changes.

The only thing that isn't great about this is if you push a bunch right in a row, sometimes it might combine to say something like "@bergerhoffer force-pushed several times, most recently from <commit-id> to <commit-id>." And that link won't give you all the changes, only the latest push's. You can sometimes finagle the URL to find the commits you want for the diff. But just letting you know this since it's not completely foolproof :)

ncbaratta commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/u.adoc:
```diff
@@ -171,7 +171,7 @@
 [[upstream-n]]
 ==== image:images/yes.png[yes] upstream (noun)
-*Description*: "Upstream" is data sent from a customer to a network service provider. Use the one-word form for the nominal form.
+*Description*: _Upstream_ is data sent from a customer to a network service provider. Use the one-word form for the nominal form.
```
Duplicate of above

IngridT1 replied:
The first is an adjective. The second is a noun.

ncbaratta replied:
I looked multiple times to find the differences! Thank you :)

IngridT1 commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:
```diff
@@ -612,7 +612,7 @@ Red Hat Directory Server conforms to LDAP standards.
 [[drl]]
 ==== image:images/yes.png[yes] DRL (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _DRL_ is an abbreviation for the Drools Rule Language, which is a file with the .drl extension. A DRL file stores technical rules as text and can be managed in the Red Hat Red Hat JBoss BRMS user interface. A DRL file contains one or more rules.
+*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _DRL_ is an abbreviation for the "Drools Rule Language", which is a file with the .drl extension. A DRL file stores technical rules as text and can be managed in the Red Hat Red Hat JBoss BRMS user interface. A DRL file contains one or more rules.
```
This looks odd to me:
"can be managed in the *Red Hat Red Hat* JBoss BRMS user interface". Are there supposed to be two Red Hats in a row like that?

sbmetz replied:
Looks to me like someone did a global change from JBoss to Red Hat JBoss without realizing this instance already had a Red Hat?

bergerhoffer replied:
Good catch! I removed the duplicate "Red Hat". 

@sbmetz if you had any comments, I don't see them. Did you still need to hit submit?

IngridT1 commented:
LGTM

sbmetz commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:
```diff
@@ -636,7 +636,7 @@ Red Hat Directory Server conforms to LDAP standards.
 [[dsl]]
 ==== image:images/yes.png[yes] DSL (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _DSL_ is an abbreviation for domain-specific language. DSL is used to create a rule language that is dedicated to your problem domain. A set of DSL definitions consists of transformations from DSL sentences to DRL constructs. These constructs let you use all of the underlying rule language and engine features. You can write rules i ...
+*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _DSL_ is an abbreviation for "domain-specific language". DSL is used to create a rule language that is dedicated to your problem domain. A set of DSL definitions consists of transformations from DSL sentences to DRL constructs. These constructs let you use all of the underlying rule language and engine features. You can write rules ...
```
No -- DRL = Drools Rule Language, business rules that go in .drl text files. There's a DRL entry above DSL -- but given this potential confusion (the similarity of the acronyms) shall we expand DRL or add it to the See also?

ncbaratta replied:
I think so.

bergerhoffer replied:
I expanded it to say "which are translated into Drools Rule Language (DRL) files." Let me know if that works!

ncbaratta replied:
Lgtm!

sbmetz commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:
```diff
@@ -1010,7 +1010,7 @@ With a _cross-forest trust_ between an Active Directory (AD) forest root domain
 [[crush]]
 ==== image:images/yes.png[yes] CRUSH (noun)
-*Description*: In Red Hat Ceph Storage, _CRUSH_ is an abbreviation for Controlled Replication Under Scalable Hashing. This is the mechanism of data distribution in a Ceph cluster. Use all capital letters when referring to "CRUSH". Do not expand, only when explaining what the abbreviation means. See the https://access.redhat.com/documentation/en/red-hat-ceph-storage/2/single/architecture-guide#cru ...
+*Description*: In Red Hat Ceph Storage, _CRUSH_ is an abbreviation for "Controlled Replication Under Scalable Hashing". This is the mechanism of data distribution in a Ceph cluster. Use all capital letters when referring to "CRUSH". Do not expand, only when explaining what the abbreviation means. See the https://access.redhat.com/documentation/en/red-hat-ceph-storage/2/single/architecture-guide#c ...
```
Do we need the link/can we just omit the last line? We don't usually have links to product doc, and this is a good example of the maintenance issue.

bergerhoffer replied:
Thanks for pointing out the 404! I agree that we can just remove it, so I'll do that now.

bergerhoffer commented:
Updated to fix other unrelated issues. Diff is here: https://github.com/redhat-documentation/supplementary-style-guide/compare/923aaf08be970e45348292fb7a49218b5153c521..d034e5732585a43df955d8d898bc717bc0c86f42

PTAL when you all can!
